kubeflow/openmpi/util.libsonnet

```diff
@@ -14,4 +14,27 @@
       [std.splitLimit(keyValue, "=", 1)[0]]: std.splitLimit(keyValue, "=", 1)[1]
       for keyValue in $.toArray(str)
     } else {},
+
+  // Convert a string to upper case.
+  upper:: function(x) {
```

Comment:
I think jsonnet has `std.asciiUpper`.  Are there any specific reason to use custom method??

Reply:
The reason for this was to be consistent with a number of other places where upper and toBool appear such as https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/util.libsonnet#L15-L22

Reply:
I think std.asciiUpper was added in a later version of jsonnet. We should get rid of our custom function once we a require a version of ksonnet with a sufficiently new version of the jsonnet library.

Please a TODO to that effect.

Reply:
It would be great to file an issue for that and reference the issue in the TODO

Reply:
I created an issue https://github.com/kubeflow/kubeflow/issues/1826 and updated commit with corresponding TODO